dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-5.1/io.openliberty.mpMetrics-5.1.feature

```diff
@@ -34,10 +34,10 @@ IBM-ShortName: mpMetrics-5.1
 Subsystem-Name: MicroProfile Metrics 5.1
 -features=io.openliberty.restHandler.internal-1.0, \
   io.openliberty.mpConfig-3.1, \
-  io.openliberty.jakarta.annotation-2.1,\
-  io.openliberty.servlet.internal-6.0, \
-  io.openliberty.mpCompatible-6.1, \
-  io.openliberty.cdi-4.0, \
+  io.openliberty.jakarta.annotation-2.1; ibm.tolerates:="3.0",\
+  io.openliberty.servlet.internal-6.0; ibm.tolerates:="6.1", \
+  io.openliberty.mpCompatible-6.1; ibm.tolerates:="7.0", \
+  io.openliberty.cdi-4.0; ibm.tolerates:="4.1", \
```

Comment:
Not sure if WLP-Platform should include microProfile-7.0 as well.

Reply:
MP Metrics is no longer part of the umbrella MP release, it is now standalone